static/js/apps/nl_interface/debug_info.tsx

```diff
@@ -18,12 +18,22 @@
  * Debug info for a single query for the NL interface
  */
 
+import { debug } from "console";
 import _ from "lodash";
 import React, { useState } from "react";
 import { Col, Row } from "reactstrap";
 
 import { DebugInfo, SVScores } from "../../types/app/nl_interface_types";
 
+const FEEDBACK_LINK =
+  "https://docs.google.com/forms/d/e/1FAIpQLSfqndIayVhN1bN5oeZT0Te-MhhBMBR1hn97Lgr77QTOpga8Iw/viewform?usp=pp_url";
```

Comment:
Suggestion: can we also add a `JSON.stringify(DebugInfo)` pre-filled answer to the form?

Reply:
I originally wanted to include debugInfo, but unfortunately it is too many characters to pass in the URL

Reply:
debug-info might be too big.  its perhaps better suited for the BT log Bo has set up?